src/test/java/emissary/test/core/junit5/ExtractionTest.java

```diff
@@ -419,4 +433,13 @@ protected void setupPayload(IBaseDataObject payload, Document doc) {
             payload.setFileType(payload.currentForm());
         }
     }
+
+    // allow the validation of att and ext counts to be skipped/not logged in tests
+    protected void setAttachmentCountValidation(boolean skipAttCountValidation) {
+        this.skipAttCountValidation = skipAttCountValidation;
+    }
+
+    protected void setExtractCountValidation(boolean skipExtractCountValidation) {
+        this.skipExtractCountValidation = skipExtractCountValidation;
+    }
```

Comment:
These parameter names are unexpected, since they're the inversion of what the method name would suggest.  Should rename to clarify
```suggestion
    protected void setAttachmentCountValidation(boolean value) {
        this.skipAttCountValidation = !value;
    }

    protected void setExtractCountValidation(boolean value) {
        this.skipExtractCountValidation = !value;
    }
```